### DISCUSSION TITLE
Flux V2 2.0 GA

### DISCUSSION THREAD

**Discussion by wsmelton** [Azure/AKS#3779] [General]
Now that 2.0.0 has been released and is GA, when run the extension with AKS get updated?

https://github.com/fluxcd/flux2/releases

**Comment by philwelz** [comment]
As Flux is an extension you are responsible for updating it. You can find the matching version of `microsoft.flux` and `flux` [here](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/extensions-release#flux-gitops)
